IRremoteESP8266.cpp

```diff
@@ -713,26 +713,35 @@ void ICACHE_FLASH_ATTR IRsend::space(unsigned long time) {
   }
 }
 
+// Calculate the period for a given frequency. (T = 1/f)
+//
+// Args:
+//   freq: Frequency in Hz.
+// Returns:
+//   nr. of uSeconds.
+uint32_t ICACHE_FLASH_ATTR IRsend::calcUSecPeriod(uint32_t hz) {
+  return (1000000UL + hz/2) / hz;  // round(1000000/hz).
```

Comment:
Oh, and just in case it wasn't obvious, this all a hack to avoid using floating point. i.e. Integer math only.